docsrc/community.rst

```diff
@@ -3,7 +3,14 @@
 Community
 =========
 
-You can get involved in the GeoNetwork project by signing up for a mailing list or finding us on Gitter and StackExchange.
+You can get involved in the GeoNetwork project by joining user forum or finding us on Gitter and StackExchange.
+
+
+User forum
+----------
+
+Join discussion on `OSGeo discourse <https://discourse.osgeo.org/c/geonetwork/55/none>`_. There are also forum for `spanish <https://discourse.osgeo.org/c/geonetwork/user-es/56>`_, `french <https://discourse.osgeo.org/c/geonetwork/user-fr/57>`_, `developers <https://discourse.osgeo.org/c/geonetwork/user-fr/58>`_  and one for the `geonetwork-ui project <https://discourse.osgeo.org/c/geonetwork/user-fr/57>`_.
```

Comment:
```suggestion
Join discussion on `OSGeo Discourse <https://discourse.osgeo.org/c/geonetwork/55/none>`_.

The forum has GeoNetwork User categories for `English <https://discourse.osgeo.org/c/geonetwork/user/54>`_, `spanish <https://discourse.osgeo.org/c/geonetwork/user-es/56>`_, `french <https://discourse.osgeo.org/c/geonetwork/user-fr/57>`_ discussion. 

There is a category for the `geonetwork-ui project <https://discourse.osgeo.org/c/geonetwork/user-fr/57>`_.
```.

For programmers there is a category `geonetwork developer <https://discourse.osgeo.org/c/geonetwork/user-fr/58>`_  discussion.

Reply:
The top-level category does not accept posts, need an entry for English user category.